streams/src/test/java/org/apache/kafka/streams/integration/SuppressionIntegrationTest.java

```diff
@@ -479,6 +557,14 @@ private void produceSynchronously(final String topic, final List<KeyValueTimesta
         }
     }
 
+    private void verifyErrorShutdown(final KafkaStreams driver) throws InterruptedException {
```

Comment:
nit: this can be simplified to:
`TestUtils.waitForCondition(() -> driver.state() == KafkaStreams.State.ERROR, TIMEOUT_MS, "Streams didn't shutdown in error state");`